### PR TITLE
[akali] Add new port

### PR DESCRIPTION
**Pull request by winsoft666**
new port named akali

### CODE REVIEW

File: ports/akali/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+	PREFER_NINJA
+    OPTIONS
+        -DBUILD_TESTS:BOOL=OFF
+)
+
+vcpkg_fail_port_install(ON_TARGET "x64-uwp")
+vcpkg_fail_port_install(ON_TARGET "arm-uwp")
+vcpkg_fail_port_install(ON_TARGET "arm64-windows")
```
Comment (PhoebeHui) [inline ports/akali/portfile.cmake]: It doesn't support uwp, arm and arm64, right? if so, you can use simply use  'vcpkg_fail_port_install(ON_ARCH "arm" "arm64" ON_TARGET  "UWP")' instead.

**Review by PhoebeHui — CHANGES_REQUESTED** [review]

**Comment by winsoft666** [comment]
any other problem?

File: ports/akali/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+vcpkg_install_cmake()
+
+if(EXISTS ${CURRENT_PACKAGES_DIR}/lib/cmake/akali)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/akali)
+elseif(EXISTS ${CURRENT_PACKAGES_DIR}/share/akali)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH share/akali)
+endif()
+
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/akali RENAME copyright)
```
Comment (PhoebeHui) [inline ports/akali/portfile.cmake]: We usually put 'file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)' at the end of portfile.cmake, also use ${PORT} instead.

**Review by PhoebeHui — APPROVED** [review]

File: ports/akali/portfile.cmake (hunk 2 above)
Comment (winsoft666) [reply ports/akali/portfile.cmake]: thanks, I have updated portfile.cmake.

**Pull request merged**